hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/BlockStreamManagerImpl.java

```diff
@@ -103,6 +105,9 @@ public class BlockStreamManagerImpl implements BlockStreamManager {
     private Instant lastHandleTime = Instant.EPOCH;
     // All this state is scoped to producing the current block
     private long blockNumber;
+    private int eventIndex = 0;
```

Comment:
Should this by default be -1 or something invalid, to indicate thats not updated?